functionaltests/main_test.go

```diff
@@ -116,3 +116,14 @@ func regionFrom(target string) string {
 		panic("target value is not accepted")
 	}
 }
+
+func endpointFrom(target string) string {
+	switch target {
+	case "qa":
+		return "https://public-api.qa.cld.elstc.co"
+	case "production":
+		return "https://api.elastic-cloud.com"
```

Comment:
Won't this panic? https://github.com/elastic/apm-server/blob/4997d07805bcc55c64a2d58cffec75e73c80ce54/.github/workflows/functional-tests.yml#L28

Reply:
yes. I updated it in [e528ec5](https://github.com/elastic/apm-server/pull/15356/commits/e528ec5ff28115f7fbcc00a1a33390f4481615a4) and also moved to using constants to reduce the chance of this occurring. In parallel I'm discussing how to make this clearer in https://github.com/elastic/apm-server/pull/14935#discussion_r1906393191